src/components/table-row-data.tsx

```diff
@@ -8,18 +9,20 @@ interface TableRowDataProps {
     itemId: string;
     itemName: string;
     imageUrl: string;
-    itemSector: string;
-    itemType: string; 
+    itemGeneric1: string;
+    itemGeneric2: string;
+    itemType: string;
+    LinkedInButton?: ReactNode;
     handleDelete: () => void;
-  }
+}
 
-export default function TableRowData({itemId, itemName, imageUrl, itemSector, itemType, handleDelete }: TableRowDataProps) {
+export default function TableRowData({ itemId, itemName, imageUrl, itemGeneric1, itemGeneric2, itemType, handleDelete, LinkedInButton }: TableRowDataProps) {
```

Comment:
@BelenA25 why you use itemGeneric as a variable name?

Reply:
I use itemGeneric to keep the component flexible and adaptable for different tables with varying database field names.